.changeset/calm-horses-feel.md

```diff
@@ -0,0 +1,11 @@
+---
+"undefined": minor
+"@fluidframework/core-interfaces": minor
+"@fluidframework/tree": minor
+---
+---
+"section": other
+---
+
```

Comment:
Can you add a short description as a heading? It can just be a phrase, and it doesn't need punctuation at the end.